### PR TITLE
matplotlib->altair changes

### PR DESCRIPTION
**Pull request by KhalidAlt**

### CODE REVIEW

File: The Annotated Transformer.md [hunk 1]
```diff
@@ -261,8 +256,19 @@ def subsequent_mask(size):
 > Below the attention mask shows the position each tgt word (row) is allowed to look at (column). Words are blocked for attending to future words during training.
 
 ```python
-plt.figure(figsize=(5,5))
-plt.imshow(subsequent_mask(20)[0])
+LS_data=pd.concat([pd.DataFrame({"Subsequent Mask":subsequent_mask(20)[0][x,y].flatten(),
```
Comment (austinvhuang) [inline The Annotated Transformer.md:259]: minor nit throughout - space around assignment per https://www.python.org/dev/peps/pep-0008/#other-recommendations

File: The Annotated Transformer.md [hunk 2]
```diff
@@ -616,15 +632,37 @@ def get_std_opt(model):
...
         NoamOpt(512, 1, 8000, None),
         NoamOpt(256, 1, 4000, None)]
-plt.plot(torch.arange(1, 20000), [[opt.rate(i) for opt in opts] for i in range(1, 20000)])
-plt.legend(["512:4000", "512:8000", "256:4000"])
-plt.gca().set_xlabel("Step")
-plt.gca().set_ylabel("Learning Rate")
-None
+
+
+opts = [NoamOpt(512, 1, 4000, None),
```
Comment (austinvhuang) [inline The Annotated Transformer.md:644]: may ask you for a follow-up merge fix after optimizer changes https://github.com/austinvhuang/annotated-transformer/pull/4

**Review by austinvhuang — CHANGES_REQUESTED** [review]
Overall LGTM. May ask for some fixes depending on what breaks with https://github.com/austinvhuang/annotated-transformer/pull/4 as well as updates to dataloaders. Can you just fix the spaces around assignments (https://www.python.org/dev/peps/pep-0008/#other-recommendations) and we'll go ahead and merge this? 

Thanks!
